test/tests-gemini-bu-rot/app.toml

```diff
@@ -3,6 +3,7 @@ target = "thumbv8m.main-none-eabihf"
 board = "gemini-bu-rot-1"
 chip = "../../chips/lpc55"
 stacksize = 1400
+image-names = ["a", "b"]
```

Comment:
Deliberately included in this PR?

Reply:
Yes. This PR grew a bit and I discovered this was missing when I went to run the tests.